crates/egui_kittest/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "egui_kittest"
+edition.workspace = true
+license.workspace = true
+rust-version.workspace = true
+version.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+wgpu = ["dep:egui-wgpu", "dep:pollster", "dep:image"]
+snapshot = ["dep:dify"]
+
+default = ["image?/png"]
```

Comment:
All crates in this repository uses [document-features](https://crates.io/crates/document-features) to document their features. Please do the same here!